README.md

```diff
@@ -21,7 +21,7 @@ You can import in Deno via:
 <!-- x-release-please-start-version -->
 
 ```ts
-import OpenAI from 'https://raw.githubusercontent.com/openai/openai-node/v4.13.0-deno/mod.ts';
+import OpenAI from 'https://raw.githubusercontent.com/openai/openai-node/v4.14.0-deno/mod.ts';
```

Comment:
@RobertCraigie should this be `4.14.1` given that's the release number for this PR?

Reply:
Sorry, let me change this to use esm.sh…

Reply:
Upon further investigation, we're just going to remove "direct" Deno support and have users `import OpenAI from "npm:openai"` instead, which is much less error-prone, since [Deno Deploy launched support for this last month](https://deno.com/blog/npm-on-deno-deploy).